### PR TITLE
#125 Open Handling Unit Editor

### PR DESCRIPTION
**Pull request by cristinastefan90**
https://github.com/metasfresh/metasfresh-e2e/issues/125

### CODE REVIEW

File: cypress/integration/sales/sales_order_to_shipment_to_invoice.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ describe('Create Sales order', function() {
       .get('li')
       .eq('1')
       .click({ force: true });
-    // cy.wait(8000);
+    cy.wait(8000);
     cy.get('.btn-header.side-panel-toggle').click({ force: true });
```
Comment (siemiatj) [inline cypress/integration/sales/sales_order_to_shipment_to_invoice.js]: won't adding a timeout option here work ? Not a fan of explicitly waiting for something to happen.

https://docs.cypress.io/api/commands/click.html#Arguments

Comment (cristinastefan90) [reply cypress/integration/sales/sales_order_to_shipment_to_invoice.js]: tried,not worked.

**Comment by siemiatj** [comment]
@cristinastefan90 I'll try rerunning but it failed the first time https://jenkins.metasfresh.com/job/ops/job/run_e2e_tests/378/

File: cypress.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "baseUrl_comment1":  "baseUrl can be overridden by setting the CYPRESS_baseUrl environment variable",
   "baseUrl_comment2":  "Example: CYPRESS_baseUrl=http://192.168.99.100:30080 npm run cypress:open",
   "baseUrl_comment3":  "See https://docs.cypress.io/guides/guides/environment-variables.html#We-can-move-this-into-an-environment-variable",
-  "baseUrl": "http://localhost:3000",
+  "baseUrl": "https://localhost:3000",
```
[the remark this answers is not in the document]
Comment (cristinastefan90) [reply cypress.json]: my bad, the https part was from a dev instance I was using and when I have put localhost back I forgot to modify it entirely:)

**Pull request merged**